MAUI/Popup/styles.md

```diff
@@ -0,0 +1,300 @@
+    // Setting the AppearanceMode as TwoButton
+    popup.ShowFooter = true;
+    popup.AppearanceMode = Syncfusion.Maui.Popup.PopupButtonAppearanceMode.TwoButton;
+
+    // Footer customization
+    popup.PopupStyle.FooterBackground = Color.LightGray;
+    popup.PopupStyle.AcceptButtonBackground = Color.FromRgb(105, 105, 105);
+    popup.PopupStyle.AcceptButtonTextColor = Color.White;
+    popup.PopupStyle.DeclineButtonBackground = Color.FromRgb(105, 105, 105);
+    popup.PopupStyle.DeclineButtonTextColor = Color.White;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup with footer customization](/Images/styles/maui-popup-with-footer-customization.png)
+
+## Stroke customization
+
+The SfPopup allows customizing the stroke appearance using the following properties.
+
+<table>
+<tr>
+<th> Property </th>
+<th> Description </th>
+</tr>
+<tr>
+<td> Stroke </td>
+<td> Gets or sets the stroke color for the PopupView.</td>
+</tr>
+<tr>
+<td> StrokeThickness </td>
+<td> Gets or sets the stroke thickness for the PopupView.</td>
+</tr>
+<tr>
+<td> CornerRadius </td>
+<td> Gets or sets the corner radius for the PopupView.</td>
+</tr>
+</table>
+
+Refer to the following code example for customizing the stroke elements.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3 4 5" %}
+<sfPopup:SfPopup x:Name="popup">
+    <sfPopup:SfPopup.PopupStyle>
+        <sfPopup:PopupStyle Stroke="LightBlue"
+                            StrokeThickness="3"
+                            CornerRadius="5" />
+    </sfPopup:SfPopup.PopupStyle>
+<sfPopup:SfPopup>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="4 5 6" %}
+public MainPage()
+{
+    InitializeComponent();
+    popup.PopupStyle.Stroke = Colors.LightBlue;
+    popup.PopupStyle.StrokeThickness = 3;
+    popup.PopupStyle.CornerRadius = 5;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup with stroke customization](/Images/styles/maui-popup-with-stroke-customization.png)
+
+## Styling overlay background
+
+The SfPopup allows to customize the background color of overlay using the `SfPopup.PopupStyle.OverlayColor` property.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3" %}
+        <sfPopup:SfPopup x:Name="popup">
+            <sfPopup:SfPopup.PopupStyle>
+                    <sfPopup:PopupStyle OverlayColor="LightPink" />
+            </sfPopup:Sfpopup.PopupStyle>
+        </sfPopup:SfPopup>
+{% endhighlight %}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="3" %}
+public MainPage()
+{
+    InitializeComponent();
+    popup.PopupStyle.OverlayColor = Colors.LightPink;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup with overlay customization](/Images/styles/maui-popup-with-overlay-customization.png)
+
+### Blurred background
+
+The SfPopup allows to blur the background using `SfPopup.OverlayMode` and `SfPopup.PopupStyle.BlurIntensity` properties, respectively.
```

Comment:
The SfPopup allows blurring of the background using the `SfPopup.OverlayMode` and `SfPopup.PopupStyle.BlurIntensity` properties respectively.

Reply:
modified